lib/beaker-pe/install/pe_utils.rb

```diff
@@ -899,7 +899,7 @@ def do_higgs_install host, opts
 
           #wait for output to host['higgs_file']
           #we're all done when we find this line in the PE installation log
-          higgs_re = /Please\s+go\s+to\s+https:\/\/.*\s+in\s+your\s+browser\s+to\s+continue\s+installation/m
+          higgs_re = /o\s+to\s+https:\/\/.*\s+in\s+your\s+browser\s+to\s+continue\s+installation/m
```

Comment:
Ah, yes this should fix the problem. But doesn't it break compatibility with older versions? Perhaps you should have it use the `version_is_less` to determine which string to grep for?